pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[tool.poetry]
+name = "django-serializable-model"
+version = "0.0.5"
+description = "Django classes to make your models, managers, and querysets serializable, with built-in support for related objects in ~100 LoC"
```

Comment:
all the comments and stuff make this around ~150 LoC now. Maybe re-word to sLoC? (here and in `setup.py` ofc)

Reply:
Changed everything to ~150 LoC except in the README Backstory as that's referring to the original version. Am going to add that commit in a separate PR or push as it's not directly related to these changes (and it was >100 LoC before already)

Reply:
And decided to throw it in here anyway and add contributing docs (ish) and a publish commit here too -- might as well throw it all it one as it's already huge (welp) and this is going to be released as soon as it's merged